test/testcafe/framework/page-objects/SymantecAuthenticatorPageObject.js

```diff
@@ -10,10 +11,16 @@ export default class SymantecAuthenticatorPageObject extends BasePageObject {
   }
 
   getPageSubtitle() {
+    if (userVariables.v3) {
+      return this.getFormSubtitle();
+    }
     return this.form.getElement('.okta-form-subtitle').textContent;
   }
 
-  submit() {
+  submit(buttonName) {
+    if (userVariables.v3) {
+      return this.form.clickButton(buttonName);
+    }
     return this.form.clickSaveButton();
   }
```

Comment:
Depeneding on how the button is set up in v2, I think you can just return `this.form.clickButton(buttonName);` and it may work for both v2 and v3.

Reply:
How about let's try not change v2 related code if possible, since it may introduce unexpected false positive tests.